tox.ini

```diff
@@ -16,7 +16,7 @@ commands=
 
 [testenv:tests]
 commands=
-    pytest -v --doctest-modules iotlabsshcli --cov=iotlabsshcli
+    pytest --doctest-modules iotlabsshcli --cov=iotlabsshcli
```

Comment:
clutters my screen when looking for summary of coverage.  more compact report.  and tests are green anyway :)